### ISSUE TITLE
wether to consider support "limit xx offset xx" statement

### ISSUE THREAD

**Issue by yourmain** [prestosql/presto#4335]
For now only support `offset xx limit xx`,
wether to consuder support `limit xx offset xx`?

**Comment by findepi** [comment]
The ANSI SQL syntax is:
```
OFFSET n [ROWS] FETCH NEXT m [ROWS] ONLY
```

(I am simplifying, there are some more variations there).


For user's convenience, and also because we already supported the non-standard `LIMIT`,
we decided to support the following:
```
OFFSET n LIMIT m
```

The `LIMIT m OFFSET n` syntax can be read both ways:

- one can read this as "limit to m rows" _then_ "discard first n rows", i.e. returning rows `[m-n+1 ... m]`
- one can also expect this to behave as in some other databases, ie same as `OFFSET n LIMIT m`, ie returning rows `[n+1 ... n+m]`

To avoid this ambiguity, we decided not to support it.
